packages/prosemirror-lwdita/src/github-integration/github.plugin.ts

```diff
@@ -75,9 +75,12 @@ export const transformGitHubDocumentToProsemirrorJson = async (rawDocument: stri
  * @param branch - The branch from which to fetch the document.
  * @returns A promise that resolves to the transformed ProseMirror JSON document.
  */
-export const fetchAndTransform = async (config: Config, ghrepo: string, source: string, branch: string) => {
+export const fetchAndTransform = async (config: Config, ghrepo: string, source: string, branch: string, referer: string) => {
   const rawDoc = await fetchRawDocumentFromGitHub(config, ghrepo, source, branch);
-  const jsonDoc = await transformGitHubDocumentToProsemirrorJson(rawDoc);
+  // apply changes to the raw xml document and update all links to point to the base URL of the referer
+  const baseUrl = referer.split('/').slice(0, -1).join('/');
+  const updatedDoc = rawDoc.replace(/href="([^"]+)"/g, `href="${baseUrl}/$1"`);
```

Comment:
This looks like it just replaces all href - what if a href points to an external image hosted elsewhere? We need to be more selective to only resolve relative URL - we should again use the URL API please.

Reply:
Fixed in https://github.com/evolvedbinary/prosemirror-lwdita/pull/479/commits/2ab2b7eda37f3205521c2df2f7d86c9407227173

Reply:
@marmoure Aren't images specified with `src` and not `href`?